index.html

```diff
@@ -162,7 +162,9 @@ <h3 class="participant-name">Stefan Bache</h3>
   </div>
   <div class="further-col">
     <div class="participant-row">
-        <a href="http://ropensci.org/unconf_nomination">Please nominate yourself if you'd like to attend</a>
+        <a href="http://ropensci.org/unconf_nomination">Please nominate yourself if you'd like to attend</a>. 
```

Comment:
I really like this idea, though it will be hard for people to know who we have on our list. But it might be a good way of picking up people who will all naturally come with someone to vouch for them, and might be a way of connecting with previous unconf attendees.